backend/src/main/java/pl/edu/agh/api/controller/ActorsController.java

```diff
@@ -0,0 +1,28 @@
+package pl.edu.agh.api.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import pl.edu.agh.api.actors.Fighter;
+import pl.edu.agh.api.services.FighterQueryService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/actors")
+public class ActorsController {
+    @Autowired
+    private FighterQueryService fighterQueryService;
```

Comment:
It is better to autowire beans through constructor. For example, annotate ActorsController with @RequiredArgsConstructor and remove @Autowired from fighterQueryService. It should work fine and we would avoid autowiring through parameter. For more details see: https://stackoverflow.com/questions/40620000/spring-autowire-on-properties-vs-constructor

Reply:
Also consider adding final to all beans parameters, which are autowired. I won't rewrite this comment so just to clarify this comment relates to all code fragments where autowire through parameter occurs.